assets/templates/layout.html

```diff
@@ -112,23 +117,210 @@ <h3>Layouts</h3>
           </nav>
         </aside>
         <div class="dpsg-content-wrap">
-          <section class="dpsg-content-module">
-            <h3>Colores</h3>
+          <section class="dpsg-content-intro">
+            <div class="dpsg-blockgrey">
+              <h3>Bienvenido a Doppler Styles Guides</h3>
+              <p>
+                UI Styles Guides en vivo. Esta biblioteca está destinada a ser
+                un lenguaje visual de Doppler, para centralizar todas las marcas
+                en un solo lugar para todos nuestros productos, reutilización de
+                código y esfuerzo.
+              </p>
+              <p>
+                También permite desacoplar la interfaz de usuario de la lógica
+                de Front End y trabajar de una manera más ordenada.
+              </p>
+              <div class="dpsg-techs">
+                <div class="dpsg-icontech">
+                  <img src="img/css3.svg" alt="" />
+                </div>
+                <div class="dpsg-icontech">
+                  <img src="img/html5.svg" alt="" />
+                </div>
+                <div class="dpsg-icontech">
+                  <img src="img/jquery.svg" alt="" />
+                </div>
+              </div>
+            </div>
+            <div class="dpsg-intropic"></div>
+          </section>
+          <section class="dpsg-howtouse-content">
+            <h3>¿Cómo puedo usar esta biblioteca en mi proyecto?</h3>
+            <div class="dpsg-wrapper-text">
+              <div class="dpsg-boxtext">
+                <p>
+                  Simplemente incluye el archivo <strong>css</strong> y
+                  <strong>js</strong> en tu <strong>html</strong>.
+                </p>
+                <p>
+                  Verifique la versión correcta en la pestaña de versiones de
```

Comment:
If we are using neutral spanish here should be `Verifica` instead of `Verifique`